Portfolio Set up and Mobile version

BenjaminSemah opened the pull request:
**In this project, I did the following;**

-Added a header.
-Added a headline section with my name and bio.
-Added a skills section with languages, frameworks I work with.
-Added underline hover effect to the links.

BenjaminSemah committed:
Add linters for HTML and CSS

BenjaminSemah committed:
Add markup for Homepage

BenjaminSemah committed:
Add stylesheet for Homepage

BenjaminSemah committed:
Updated stylesheet for Homepage

BenjaminSemah committed:
Update README.md file

mmsesay commented on Index.html:
```diff
@@ -0,0 +1,128 @@
+        <div class="logo">
+            <a class="myLogo" href="#">My Logo</a>
+        </div>
+        <div class="navIcon">
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
```
- [ ] Good job in placing the hamburger button to the right. But we can try to replace the `div` tag with a `nav` tag.

mmsesay commented on Index.html:
```diff
@@ -0,0 +1,128 @@
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
+    </header>
+    <img id="whiteEllipse" src="/assets/ellipseHeader.svg" alt="ellipse">      
```
- [ ] This is really good but we can try to set the image as a background image inside your CSS file instead of directly setting the image inside the `img` tag

mmsesay commented on Index.html:
```diff
@@ -0,0 +1,128 @@
+                    <li>
+                        <a href="#">Gitlab</a>
+                    </li>
+                    <li>
+                        <a href="#">Terminal</a>
+                    </li>
+                </ul>
+            </div>
+        <img class="line" src="/assets/Vector 4.svg" alt="horizontal line">
+        </div>
+    </section>
+    <section class="socials">
+        <ul class="profiles">
+            <li>
+                <a href="#">
+                    <img src="/assets/github.svg" alt="GitHub Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="linkedIn" src="/assets/linkedinn.png" alt="LinkedIn Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/angellist.svg" alt="Angellist Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/Twitter.svg" alt="Twitter Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/medium.svg" alt="Medium Logo">
+                </a>
+            </li>
+        </ul>
+    </section>
```
- [ ] It's good you added these lines of code. But you did go the extra mile than stated for this project.
You are only required to work on the toolbar and the headline section. Let's try to remove these lines of code.

mmsesay commented on Index.html:
```diff
@@ -0,0 +1,128 @@
+        </div>
+    </header>
+    <img id="whiteEllipse" src="/assets/ellipseHeader.svg" alt="ellipse">      
+    <section class="headline">    
+        <div class="nameTitle">
+            <h1 id="myName">Benjamin Semah</h1>
+            <h2 id="myTitle">Software Developer</h2>
+        </div>
+        <div id="bio">
+            <p id="shortBio">
+                I can help you build a product, feature or website.
+                Look through some of my work and experience! 
+                If you like what you see and have a project you need coded,  
+                don’t hestiate to contact me.
+            </p>
+        </div>
+    </section>
```
- [ ] Good one for setting this headline section. But it is not well aligned to the centre of the page. I believe it's the image pushing it a bit to the right. That's why I suggested above, you try setting the image as a background image.

- [ ] Secondly, the images are not rendering as expected. Please take a look at your design to see the point am making here. Please do not forget to also check that you use the right fonts, font size and colour for the headline contents.

Your Design              |
:--------------------------:|
![Your design](https://user-images.githubusercontent.com/39523312/143461564-541be63e-1bdb-4a3e-9601-ed6558b9932b.png)

mmsesay reviewed (changes requested):
Hi @BenjaminSemah,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mmsesay commented:
- [ ] Your project does have some failing linters. Please try to fix them also and ensure they all pass before submitting for another review.

BenjaminSemah committed:
Fix errors in stylesheet for Homepage

BenjaminSemah committed:
Update README.md

I have changed the app screenshot in the README.md file.

BenjaminSemah committed:
Update stylesheet for Homepage

BenjaminSemah commented:
**Fix errors on Homepage**

- Changed div tag to nav tag for hamburger menu.
- Used ellipse as a background image in headline section.
- Headline section properly aligned to the center.
- Deleted skills section in homepage.
- Linters errors checked and fixed.

alonsofalconi commented on Index.html:
```diff
@@ -0,0 +1,35 @@
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a class="myLogo" href="#">My Logo</a>
+        </nav>
+        <div class="navIcon">
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
```
- [ ] I think what the previous reviewer suggested was to use the `<nav>` tag for the image since this is the image for the "navigation" links. I would suggest updating this so your structure looks like this :raised_hands:
```
<header>
  <a class="myLogo" href="#">My Logo</a>
  <nav class="navIcon">
    <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
  </nav>
</header>
```

alonsofalconi commented on style.css:
```diff
@@ -0,0 +1,69 @@
+}
+
+.headline {
+  background-image: url(/assets/ellipseHeader.svg);
```
- [ ] There is an issue with your background because it's not showing. I think it is because of the `/`, you could try removing it or even better adding a `.` before so it look like this: `background-image: url(./assets/ellipseHeader.svg);`

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/143484242-c043208b-68c1-47e2-b47e-6481b2123098.png)

  With the `.` before:
  ![image](https://user-images.githubusercontent.com/81522435/143484360-eee6ced8-db31-49ea-8e29-b3a82d8e3590.png)
  ![image](https://user-images.githubusercontent.com/81522435/143484384-a9960b1d-6a8b-4ec2-b1eb-477fea712e82.png)

alonsofalconi reviewed (changes requested):
Hi @BenjaminSemah,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## To highlight :dart:
- Well documented README :heavy_check_mark:
- Descriptive PR title and description :heavy_check_mark:
- Your design looks great :fire:
- Meaningful commit messages :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenjaminSemah committed:
Fix errors on Homepage

I have appropiately used the nav tag for the hamburger menu. I have also fixed the url for background image.

BenjaminSemah commented:
**Fix errors on Homepage**
- Used nav tag properly for hamburger menu.
- Corrected URL for background image.

alonsofalconi reviewed (approved):
Hi @BenjaminSemah,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media2.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
